### ISSUE TITLE
Respect Teams

### ISSUE THREAD

**Issue by breon** [Incandescent-Turtle/FriendHighlighter#10]
Is there a way to respect team colors automatically? We host events with many players and I would have to add each team member and their team color one at a time. Also, is it possible to have projectiles respect the team color of the person that shot it? The second one is less important.

**Comment by Mqnchy** [comment]
team colors would be veryyy nice to have :P

**Comment by Incandescent-Turtle** [comment]
This has been added in the development branch, and is stable and working. This will be included in the next update. Not the projectile part though.

**Comment by Incandescent-Turtle** [comment]
Upon looking into it, arrows in minecraft do not carry data about their thrower/shooter, so implementing that part with a clientside mod such as this would be difficult and not completely accurate. so that part most likely will never be added to this mod by me

**Comment by breon** [comment]
Thanks for the updates!

I'm surprised that Minecraft doesn't expose who the projection belongs to. They'd have to know which player shot it so they could attribute the damage/kill to the correct player.  

Oh well. Maybe one day.

**Comment by Incandescent-Turtle** [comment]
@breon you're right....that is surprising. that indeed makes no sense. projectiles have an owner, and i am checking rn to see if that would work for our purposes :)

**Comment by Incandescent-Turtle** [comment]
yep i have now implemented projectile highlighting! commit can be seen here: https://github.com/Incandescent-Turtle/FriendHighlighter/commit/7d8ac2bd682fec5a92c3b3c9179decf0039ac6db

looks bug free! thanks for suggesting it & telling me it was, in fact, possible. this feature will be included in the next update (coming soon). i will keep this issue open until the update is out and these two suggestions are stable + released

the way that I have implemented it allows projectiles to be highlighted in the color of the shooter, whether that be the team color, or the color you've set the shooter's highlight to! it even works retroactively on already shot arrows.

**Comment by Incandescent-Turtle** [comment]
these features have been implemented on the beta v2 release on github (https://github.com/Incandescent-Turtle/FriendHighlighter/releases/tag/v2.0.0beta-1.20.1) for 1.20.1. once testing has been done and bug fixes are applied, the full release of the update will be released for 1.19.x, 1.20.x, and 1.21.x
